pyqmix_backend/app.py

```diff
@@ -166,18 +171,24 @@ def is_config_set_up():
         else:
             return True
 
-
-def set_up_config(config_dir):
+def set_up_config(config_name):
 
     if app.config['test_session']:
         print(f'Pump configuration is set up using '
-              f'config path: {config_dir}')
+              f'config name: {config_name}')
     else:
-        config.set_qmix_config_dir(config_dir)
+        # Config path
+        default_config_path = 'C:\\Users\\Public\\Documents\\QmixElements\\Projects\\default_project\\Configurations'
```

Comment:
This is fine, however the -- in my opinion -- cleanest way would be to do:
```python
 default_config_path = os.path.normpath('C:\\Users\\Public\\Documents\\QmixElements\\Projects\\default_project\\Configurations')
```
(Or use forward slashes in the path. In any case, `normpath()` ensures that the path is "clean")